### PR TITLE
do not review or comment or anything, testing the reviews page.

### PR DESCRIPTION
**Pull request by jdhollen**
**Related issues**: N/A

### CODE REVIEW

**Comment by siggisim** [comment]
Whoops

File: enterprise/server/githubapp/githubapp.go [hunk 1]
```diff
@@ -1452,13 +1452,17 @@ func (a *GitHubApp) GetGithubPullRequest(ctx context.Context, req *ghpb.GetGithu
 	}
 	resp := &ghpb.GetGithubPullRequestResponse{
 		Outgoing: []*ghpb.PullRequest{},
-		Incoming: []*ghpb.PullRequest{},
+		Incoming: []*ghpb.PullRequest{},asdfasd
 	}
 	for _, i := range outgoing.Issues {
-		resp.Outgoing = append(resp.Outgoing, prs[i.GetNodeID()])
```
Comment (jdhollen) [inline enterprise/server/githubapp/githubapp.go:1458]: good evening i'm testing comments again.

Comment (siggisim) [reply enterprise/server/githubapp/githubapp.go:1458]: good evening

File: enterprise/server/githubapp/githubapp.go [hunk 2]
```diff
@@ -1452,13 +1452,17 @@ func (a *GitHubApp) GetGithubPullRequest(ctx context.Context, req *ghpb.GetGithu
 	}
 	resp := &ghpb.GetGithubPullRequestResponse{
 		Outgoing: []*ghpb.PullRequest{},
```
Comment (jdhollen) [inline enterprise/server/githubapp/githubapp.go:1454]: r

File: enterprise/server/githubapp/githubapp.go (hunk 1 above)
Comment (jdhollen) [reply enterprise/server/githubapp/githubapp.go:1458]: Acknowledged.

File: enterprise/server/githubapp/githubapp.go (hunk 2 above)
Comment (jdhollen) [reply enterprise/server/githubapp/githubapp.go:1454]: hello, again

Comment (siggisim) [reply enterprise/server/githubapp/githubapp.go:1454]: hello darkness my old friend

Comment (jdhollen) [reply enterprise/server/githubapp/githubapp.go:1454]: Acknowledged.

**Review by jdhollen — COMMENTED** [review]
I wrote this review in the BuildBuddy code review UI alpha!

**Review by jdhollen — COMMENTED** [review]
More testing..

File: enterprise/server/githubapp/githubapp.go [hunk 3]
```diff
@@ -1452,13 +1452,17 @@ func (a *GitHubApp) GetGithubPullRequest(ctx context.Context, req *ghpb.GetGithu
...
-		Incoming: []*ghpb.PullRequest{},
+		Incoming: []*ghpb.PullRequest{},asdfasd
 	}
 	for _, i := range outgoing.Issues {
```
Comment (jdhollen) [inline enterprise/server/githubapp/githubapp.go:1457]: testing, again

**Review by jdhollen — COMMENTED** [review]
See comments

**Pull request closed without merging**